src/Controllers/Apiv2Controller.php

```diff
@@ -360,7 +360,7 @@ private function getSubModel(?ApiSubModels $submodel): RestInterface
                 ApiSubModels::Containers => LinksFactory::getContainersLinks($this->Model, $this->subId),
                 ApiSubModels::ExperimentsLinks => LinksFactory::getExperimentsLinks($this->Model, $this->subId),
                 ApiSubModels::Events => new Scheduler($this->Model, $this->subId),
-                ApiSubModels::Compounds => LinksFactory::getCompoundsLinks($this->Model, $this->subId),
+                ApiSubModels::CompoundsLinks => LinksFactory::getCompoundsLinks($this->Model, $this->subId),
```

Comment:
this is a breaking change in the API. Calls to `entity/id/compounds` won't work anymore, which is fine, if we want to homogeneize with `experiments_links` and `items_links`. But then you need to change the entity response (main json data) to include `compounds_links` instead of `compounds` so it is all homogeneous.